packages/execution-environments/src/common/endowments/crypto.test.ts

```diff
@@ -36,8 +39,12 @@ describe('Crypto endowment', () => {
   });
 
   it('returns Node.js webcrypto module', () => {
-    expect(crypto.factory()).toStrictEqual({
-      crypto: { mock: true, subtle: { mock: true } },
+    // eslint-disable-next-line jest/prefer-strict-equal
+    expect(crypto.factory()).toEqual({
```

Comment:
Why can you not use strict equal here?

Reply:
Not sure, my guess was because it isn't the same reference. It fails saying it serializes to the same string 🤷‍♂️